src/main/java/picard/sam/SamErrorMetric/CollectSamErrorMetrics.java

```diff
@@ -44,12 +43,16 @@
 import picard.cmdline.StandardOptionDefinitions;
 import picard.cmdline.programgroups.DiagnosticsAndQCProgramGroup;
 
+import java.io.BufferedWriter;
 import java.io.File;
+import java.io.FileWriter;
 import java.io.IOException;
 import java.util.*;
 import java.util.function.Supplier;
 import java.util.stream.Collectors;
 
+import static shaded.cloud_nio.com.google.common.collect.Iterables.concat;
+
 /**
  * Program to collect error metrics on bases stratified in various ways.
```

Comment:
please update the javadoc for this tool to reflect the changes that you are proposing to make to it as this no longer describes all that it does....